### PR TITLE
add sort for the policy on the sriovOperatorConfig controller

### PR DESCRIPTION
**Pull request by SchSeba**
we need to be consistent with the policy order

### CODE REVIEW

**Review by ykulazhenkov — COMMENTED** [review]
This is required  to keep daemon's DS NodeSelector stable?
Can you add a brief comment near the sorting?

**Review by zeeke — COMMENTED** [review]
LGTM

please consider adding a unit test like this
https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/6a1076858e3ca2166dcf9743961f04e5e2c48967

File: controllers/sriovoperatorconfig_controller.go [hunk 1]
```diff
@@ -107,6 +108,8 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+	// Sort the policies with priority, higher priority ones is applied later
```
Comment (adrianchiris) [inline controllers/sriovoperatorconfig_controller.go:111]: nit: please add comment why this sorting is needed

Comment (SchSeba) [reply controllers/sriovoperatorconfig_controller.go:111]: done

**Review by adrianchiris — APPROVED** [review]
LGTM. @SchSeba please rebase and we can merge

**Review by zeeke — APPROVED** [review]

**Review by zeeke — APPROVED** [review]

**Pull request merged**